R/app_server.R

```diff
@@ -171,6 +171,25 @@ app_server <- function(input, output, session) {
   }) %>%
     bindEvent(selected_pkg$decision(), selected_pkg$overall_comment_added())
   
+  
+  rpt_choices <- c(
+    "Report Author", "Report Date", "Risk Score", "Overall Comment", 
+    "Package Summary", "Maintenance Metrics", "Maintenance Comments", 
+    "Community Usage Metrics", "Community Usage Comments", "Source Explorer Comments"
+    )
+  session$userData$user_report <- reactiveValues()
+  observe({
+    req(user$name)
+
+    # retrieve user data, if it exists.  Otherwise use rpt_choices above.
+    session$userData$user_report$user_file <- system.file("report_downloads", glue::glue("report_prefs_{user$name}.txt"), package = "riskassessment")
```

Comment:
```suggestion
    session$userData$user_report$user_file <- file.path(system.file("report_downloads", package = "riskassessment"), glue::glue("report_prefs_{user$name}.txt"))
```

I am currently getting errors when trying to store preferences for the first time. The suggestion is one way to fix it.
![image](https://github.com/pharmaR/riskassessment/assets/85879620/8da7ef39-788f-4a24-a5bc-1157765604e6)